cmd/cstor-pool-mgmt/pool/v1alpha2/import.go

```diff
@@ -49,13 +58,19 @@ func Import(cspi *apis.CStorPoolInstance) (bool, error) {
 
 	klog.Infof("Importing pool %s %s", string(cspi.GetUID()), PoolName(cspi))
 	devID := pool.GetDevPathIfNotSlashDev(bdPath[0])
+	cmd := zfs.NewPoolImport().
+		WithCachefile(cspi.Spec.PoolConfig.CacheFile).
+		WithProperty("cachefile", cspi.Spec.PoolConfig.CacheFile).
+		WithDirectory(devID).
+		WithNewPool(PoolName(cspi))
+	// oldName denotes the pool name that may be present
+	// from previous version and needs to be imported with new name
+	oldName := cspi.Annotations[string(apis.OldPoolName)]
+	if oldName != "" {
+		cmd.WithPool(oldName)
```

Comment:
We should do the same changes in the below import also i.e only with the cache file(line num 83 with new changes).